src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/McpServer/McpServer-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Common", "AI", "MCP" ],
+  "identity": "Microsoft.Extensions.AI.Templates.McpServer.CSharp",
+  "name": "Local MCP Server Console App",
+  "description": "A project template for creating a local MCP server using the .NET MCP SDK.",
```

Comment:
"A project template for creating a Model Context Protocol (MCP) server using C# and the ModelContextProtocol package."